bazel/emscripten_toolchain/env.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+mkdir -p /tmp/bazel_emscripten/
+
+export ROOT_DIR=`(pwd -P)`
+export EMSCRIPTEN=${ROOT_DIR}/external/emscripten/emscripten
+
+export EMCC_WASM_BACKEND=1
+
+export EMCC_SKIP_SANITY_CHECK=1
+
+export EM_CONFIG=${ROOT_DIR}/emscripten_toolchain/emscripten_config
+export EM_CACHE=/tmp/bazel_emscripten/emscripten_cache
```

Comment:
Ideally you can remove this and have the cache default to  $EMSCRIPTEN/cache (assuming that directory is read/write)?

Reply:
But storing the cache in /tmp seems like a bad idea.. we want the cache to persist.    

How about: `export EM_CONFIG=${ROOT_DIR}/emscripten_toolchain/cache/`?